util/tracing/opt_trace.go

```diff
@@ -80,4 +82,5 @@ type LogicalRuleOptimizeTraceStep struct {
 	Reason string `json:"reason"`
 	ID     int    `json:"id"`
 	TP     string `json:"type"`
+	Index  int    `json:"index"`
```

Comment:
Why do you need index here?

Reply:
To indicate the order for executing optimize steps which we will use in visualized dashboard.